sdk/purview/purview-scanning-rest/review/purview-scanning.api.md

```diff
@@ -1613,136 +3171,139 @@ export interface ClassificationRulesCreateOrUpdateBodyParam {
 }
 
 // @public
-export interface ClassificationRulesCreateOrUpdatedefaultResponse extends HttpResponse {
+export interface ClassificationRulesCreateOrUpdateDefaultResponse extends HttpResponse {
     // (undocumented)
-    body: ErrorResponseModel;
+    body: ErrorResponseModelOutput;
     // (undocumented)
-    status: "500";
+    status: string;
+}
+
+// @public (undocumented)
+export interface ClassificationRulesCreateOrUpdateMediaTypesParam {
+    contentType?: "application/json";
 }
 
 // @public (undocumented)
-export type ClassificationRulesCreateOrUpdateParameters = ClassificationRulesCreateOrUpdateBodyParam & RequestParameters;
+export type ClassificationRulesCreateOrUpdateParameters = ClassificationRulesCreateOrUpdateMediaTypesParam & ClassificationRulesCreateOrUpdateBodyParam & RequestParameters;
 
 // @public
 export interface ClassificationRulesDelete200Response extends HttpResponse {
     // (undocumented)
-    body: ClassificationRule;
+    body: ClassificationRuleOutput;
     // (undocumented)
     status: "200";
 }
 
 // @public
 export interface ClassificationRulesDelete204Response extends HttpResponse {
-    // (undocumented)
-    body: Record<string, unknown>;
     // (undocumented)
     status: "204";
 }
 
 // @public
-export interface ClassificationRulesDeletedefaultResponse extends HttpResponse {
+export interface ClassificationRulesDeleteDefaultResponse extends HttpResponse {
     // (undocumented)
-    body: ErrorResponseModel;
+    body: ErrorResponseModelOutput;
     // (undocumented)
-    status: "500";
+    status: string;
 }
 
 // @public (undocumented)
 export type ClassificationRulesDeleteParameters = RequestParameters;
 
 // @public (undocumented)
 export interface ClassificationRulesGet {
-    delete(options?: ClassificationRulesDeleteParameters): Promise<ClassificationRulesDelete200Response | ClassificationRulesDelete204Response | ClassificationRulesDeletedefaultResponse>;
-    get(options?: ClassificationRulesGetParameters): Promise<ClassificationRulesGet200Response | ClassificationRulesGetdefaultResponse>;
-    put(options?: ClassificationRulesCreateOrUpdateParameters): Promise<ClassificationRulesCreateOrUpdate200Response | ClassificationRulesCreateOrUpdate201Response | ClassificationRulesCreateOrUpdatedefaultResponse>;
+    delete(options?: ClassificationRulesDeleteParameters): StreamableMethod<ClassificationRulesDelete200Response | ClassificationRulesDelete204Response | ClassificationRulesDeleteDefaultResponse>;
```

Comment:
this is because the previous code was generated from a very older version of codegen.